packages/selection/src/containers/SelectionContainer.js

```diff
@@ -169,6 +169,12 @@ export class SelectionContainer extends ControlledComponent {
 
   onFocusSelectionModelChange = ({ newSelection }) => {
     const focusedKey = this.indexKeyMap[newSelection];
+    const activeElement = document.activeElement;
+    const browserSelectionIndex = [...activeElement.parentNode.children].indexOf(activeElement);
```

Comment:
Another thing I just thought of is that this has the assumption that all "selectable" items are children of the parentNode. This isn't always the case. Since you can spread these render-props to a variety of areas it may or may not be in the correct placement.